vms/platformvm/txs/base_tx.go

```diff
@@ -96,3 +98,7 @@ func (tx *BaseTx) SyntacticVerify(ctx *snow.Context) error {
 		return nil
 	}
 }
+
+func (tx *BaseTx) Visit(visitor Visitor) error {
```

Comment:
possibly an overkill, but should we consider a different name for this tx. I understand it is structurally identical to the BaseTx, but I'd introduce a type alias for it and pick a name like `TransferTx` or something. It'll make life easier for downstreams I guess

Reply:
It is called `BaseTx` on the X-chain fwiw. `TransferTx` is a better name for it though

Reply:
I'm inclined to leave it, I think it's more confusing for down streams to have two tx names for the same functionality.